### PR TITLE
Docs: Add note arbitrary rollback unsupported

### PR DESCRIPTION
**Pull request by zbwright**
https://github.com/coreos/docs/pull/1026 has lost its fork. Creating this PR to address those issues.

### CODE REVIEW

**Comment by zbwright** [comment]
Slack discussion folded into the file.

File: os/manual-rollbacks.md [hunk 1]
```diff
@@ -1,12 +1,10 @@
 # Performing manual CoreOS Container Linux rollbacks
 
-As much as we all love flawless, automatic updates, there may be occasions when an update must be rolled back. This is fairly straightforward on Container Linux, once you know the right commands.
+In the event of an upgrade failure, Container Linux will automatically boot with the version on the rollback partition. Immediately after an upgrade reboot, the active version of Container Linux can be rolled back to the version installed on the rollback partition, or downgraded to the version current on any lower release channel.
```
Comment (joshix) [inline os/manual-rollbacks.md]: Explicit sentence to conclude this graf: "There is no method to downgrade to an arbitrary version number." (ref original intent of https://github.com/coreos/docs/issues/964)

File: os/manual-rollbacks.md [hunk 2]
```diff
@@ -145,7 +143,7 @@ If we take another look at the GPT tables, we'll see that the priorities have be
 
 ```
 
-Again, composing the previous two commands we get this handy one-liner to revert to the previous image.
+Composing the previous two commands produces the following command to set the currently passive partition to be active on the next boot:
```
Comment (joshix) [inline os/manual-rollbacks.md]: +100 much better

**Comment by joshix** [comment]
ref https://github.com/coreos/docs/issues/964

**Pull request merged**